templates/help/marketplace.html

```diff
@@ -0,0 +1,73 @@
+$:{TITLE("Marketplace", "Help", "/help")}
+
+<div class="content">
+  <div class="constrained">
+
+    <h3>What is the Marketplace?</h3>
+    <div class="formatted-content">
+      <p>The Marketplace is Weasyl’s Commission Search tool. You can use it to find artists who are open for commissions that match your search criteria.</p>
+    </div>
+
+    <h3>Searching for Artists</h3>
+    <div class="formatted-content">
+      <p>Marketplace currently supports the following search criteria:</p>
+
+      <ul>
+        <li>Type of Commission (e.g. badge)</li>
```

Comment:
“Commission” probably shouldn’t be capitalized here to match “range and currency” below.